src/main/java/org/opensrp/repository/postgres/mapper/xml/ProductCatalogueMapper.xml

```diff
@@ -124,7 +124,7 @@
       This element is automatically generated by MyBatis Generator, do not modify.
       This element was generated on Thu Oct 01 12:40:29 PKT 2020.
     -->
-    unique_id, product_name, json, server_version
+    unique_id, product_name, json AS product_catalogue_json, server_version
```

Comment:
Revert this and just create a custom result for query that returns both stock and catalog

Reply:
This alias was created to avoid the conflict of the same column name "json"  in both stock and product_catalogue table.